README.md

```diff
@@ -32,6 +32,27 @@ The different alerting rules in this repository are:
 
 * [**Alert Rule QuotaExceeded**](https://gitlab.cee.redhat.com/rhtap/docs/sop/-/blob/main/o11y/alert-rule-QuotaExceeded.md)
 
+### SLO Alerts
+
+SLO (Service Level Objective) alert rules are rules defined to monitor and alert when a service or system is not meeting its specified service level objectives.
+
+#### **Usage Guidelines:**
+
+   - **Label Requirement:** Apply the `slo:true` label to alerts directly associated with Service Level Objectives. These alerts typically indicate issues affecting the performance or reliability of the service.
```

Comment:
```suggestion
   - **Label Requirement:** Apply the `slo: true` label to alerts directly associated with Service Level Objectives. These alerts typically indicate issues affecting the performance or reliability of the service.
```

Reply:
I think there should be a space between `slo:` and `true` so it's consistent with what needs to be configured.

Reply:
done